packages/compiler/src/server/update-manager.ts

```diff
@@ -39,6 +39,7 @@ export class UpdateManager<T = void> {
   constructor(
     private name: string,
     log: (sl: ServerLog) => void,
+    getDelayCandidates?: () => number,
```

Comment:
getDelayCandidates imply some internal implementation, i think getDebounceDelay() will be better